modules/fedimint-ln-client/src/lib.rs

```diff
@@ -1002,9 +1012,15 @@ impl LightningClientModule {
         Ok(gateways)
     }
 
-    /// Pays a LN invoice with our available funds
+    /// Pays a LN invoice with our available funds using the supplied `gateway`
+    /// if one was provided and the invoice is not an internal one. If none is
+    /// supplied only internal payments are possible.
+    ///
+    /// The `gateway` can be acquired by calling
+    /// [`LightningClientModule::select_active_gateway`].
     pub async fn pay_bolt11_invoice<M: Serialize + MaybeSend + MaybeSync>(
         &self,
+        maybe_gateway: Option<LightningGateway>,
         invoice: Bolt11Invoice,
         extra_meta: M,
     ) -> anyhow::Result<OutgoingLightningPayment> {
```

Comment:
Backporting takes some more work here: the old fn signature has to be maintained and a new fn added with the updated signature. Otherwise we break SemVer.

Reply:
do we commit to semver?

Reply:
it made backport smaller, so fine for me

Reply:
oh right we publish on crates io